walkthroughs/week-5-tdd/project/src/test/java/com/google/sps/FindMeetingQueryTest.java

```diff
@@ -270,5 +272,149 @@ public void notEnoughRoom() {
 
     Assert.assertEquals(expected, actual);
   }
+
+  @Test
+  public void optionalAttendeeNotConsidered() {
+    // Have each person have different events. We should see three options because each person has
+    // split the restricted times.
```

Comment:
What does "each person has split the restricted times" mean?

Reply:
Basically means the events each attendee has split the whole day into 3 options for the request. I've changed the wording so that it's less confusing.